UnitConversion/AreaConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnitConversion.Base;
+
+namespace UnitConversion
+{
+    public class AreaConverter : BaseUnitConverter {
+        UnitFactors units = new UnitFactors("m") {
```

Comment:
I'm guessing this should be UnitFactors("m2")? 

I've just checked the unit conversion code, and actually it's not used (which is why your code works), but it's useful information for a reader. 

In the future it could also be used to validate UnitFactors.